package.json

```diff
@@ -7,6 +7,9 @@
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
     "big.js": "^6.2.1",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.7",
+    "enzyme-to-json": "^3.6.2",
```

Comment:
- [ ] You did a great job working on the tests 🚀 However, you wrote your test suite using `Enzyme` instead of using the required `React Testing Library`. Please consider refactoring your tests to use RTL. This is because Enzyme is no longer maintained for recent versions of React and also because RTL offers a lot more benefits with less effort. You can read more about this discussion [here](https://dev.to/wojtekmaj/enzyme-is-dead-now-what-ekl).